src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -466,6 +466,41 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
         }
 
         [Fact]
+        public async Task GetAsync_ResponseWithSetCookieHeaders_AllCookiesRead()
+        {
+            // Cookie names/values, with one of the values deliberately over 128 characters,
+            // which is the size of WinHttpHandler's initial response header buffer size, to
```

Comment:
> Then maybe this test really belongs in the System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs.

While there may not be an explicit limit in CurlHandler, I think there's value in having the test for it as well, to exercise a longer cookie, a pair of cookies, that there's no issue with the various characters used in the cookies, etc. Most importantly, I think there's value in having as many different tests as possible doing the same thing on both platforms.